scripts/types.d.ts

```diff
@@ -33,6 +34,7 @@ export interface ExampleShortHeader {
   authorIds: string[];
   previewImageUrls: string[];
   gdevelopVersion: string;
+  eventsSizeLevel: string;
```

Comment:
eventsCountLevel maybe? I'll let you choose.

Reply:
I'm thinking about `codeSizeLevel` because I don't want it be mistaked for a number.